components/camera/transformpipeline/pipeline.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/pkg/errors"
 	"go.opencensus.io/trace"
 	"go.uber.org/multierr"
+	rutils "go.viam.com/utils"
```

Comment:
rutils usually means the "go.viam.com/rdk/utils" package. Could you instead call this `goutils`?